mmpose/models/heads/heatmap_heads/heatmap_head.py

```diff
@@ -292,6 +292,8 @@ def predict(self,
         else:
             batch_heatmaps = self.forward(feats)
 
+        batch_heatmaps = self.select_output_indices(
```

Comment:
How about moving `select_output_indices` into `decode()` of `base_head.py`?

Reply:
`decode()` does not affect output heatmaps. There will be a mismatch between `pred_instance.keypoints` and `pred_fieds.heatmaps`